### ISSUE TITLE
nbdev_new fails on Windows

### ISSUE THREAD

**Issue by epogrebnyak** [fastai/nbdev#138]
Should  `nbdev_new` be able to create a project template in a new folder?

I'm running this command in on Windows and get `Access denied` error

```
> nbdev_new pkg2

Traceback (most recent call last):
  File "d:\anaconda3\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "d:\anaconda3\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "D:\Anaconda3\Scripts\nbdev_new.exe\__main__.py", line 9, in <module>
  File "d:\anaconda3\lib\site-packages\fastscript\core.py", line 73, in _f
    func(**args.__dict__)
  File "d:\anaconda3\lib\site-packages\nbdev\cli.py", line 258, in nbdev_new
    if os.path.isdir(path): shutil.rmtree(path)
  File "d:\anaconda3\lib\shutil.py", line 513, in rmtree
    return _rmtree_unsafe(path, onerror)
  File "d:\anaconda3\lib\shutil.py", line 392, in _rmtree_unsafe
    _rmtree_unsafe(fullname, onerror)
  File "d:\anaconda3\lib\shutil.py", line 392, in _rmtree_unsafe
    _rmtree_unsafe(fullname, onerror)
  File "d:\anaconda3\lib\shutil.py", line 392, in _rmtree_unsafe
    _rmtree_unsafe(fullname, onerror)
  File "d:\anaconda3\lib\shutil.py", line 397, in _rmtree_unsafe
    onerror(os.unlink, fullname, sys.exc_info())
  File "d:\anaconda3\lib\shutil.py", line 395, in _rmtree_unsafe
    os.unlink(fullname)
PermissionError: [WinError 5] Отказано в доступе: 'C:\\Users\\Евгений\\Desktop\\notebooks\\pkg2\\.git\\objects\\pack\\pack-32fff01fcfc3d5824e4097a65e36e8abd1bcd22c.idx'
```

**Comment by sgugger** [comment]
I have not tried to use nbdev on Windows so I don't know if this command works. It seems like your python process does not have permission to remove a folder, which you could do manually.

**Comment by epogrebnyak** [comment]
Maybe shutil.rmtree cannot remove a hidden file on Windows. Perhaps safer to use
`ignore_errors=True` below?

https://github.com/fastai/nbdev/blob/master/nbdev/cli.py#L258

**Comment by sgugger** [comment]
In any case, if you fall back to that it's because there has been an error.

**Comment by pete88b** [comment]
I think I have a fix for this: https://github.com/fastai/nbdev/pull/155

**Comment by sgugger** [comment]
Should be fixed by #155
